airflow/configuration.py

```diff
@@ -85,6 +85,15 @@ def run_command(command):
     return output
 
 
+def _get_config_value_from_secret_backend(config_key):
```

Comment:
Any reason why this is a function in the module, and not a method on the AirflowConfiguration class?

Reply:
Just to keep it similar to `def run_command(command):`, no other reason